README.md

```diff
@@ -8,16 +8,39 @@
 
 Shows the rendered HTML of the AsciiDoc content in current editor using <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd>.
 
-It can be activated from the editor using the <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd> key-binding and is currently enabled for `.adoc`, `.asciidoc`, `.ad`, `.asc`, `.adoc.txt`, `.txt` files.
+It is currently enabled for `.adoc`, `.asciidoc`, `.ad`, `.asc`, `.adoc.txt`, `.txt` files.
 
-**Note:** since 1.0.0, the autocompletions have been migrated to a new package: [autocomplete-asciidoc](https://atom.io/packages/autocomplete-asciidoc)
+You can toggle the preview pane, which appears to the right of the editor, using the key combination <kbd>ctrl-shift-a</kbd> or <kbd>cmd-shift-a</kbd>.
```

Comment:
I have remove shortcuts from the first sentence and move "You can toggle..." below the first sentence.